Tests/Podfile

```diff
@@ -4,7 +4,7 @@ xcodeproj 'SDWebImage Tests'
 workspace '../SDWebImage'
 
 def import_pods
-  pod 'Expecta'                       # A Matcher Framework for Objective-C/Cocoa
+  pod 'Expecta', '<=0.3.1'                    # A Matcher Framework for Objective-C/Cocoa
```

Comment:
The unit test failure has been caused by `pod update` failure because Expecta does not support iOS5 from v.0.3.2.